client/tsconfig.json

```diff
@@ -1,25 +1,26 @@
 {
-  "compilerOptions": {
-    "module": "commonjs",
-    "target": "es5",
-    "outDir": "build",
-    "noImplicitAny": true,
-    "noUnusedParameters": true,
-    "noUnusedLocals": true,
-    "preserveConstEnums": true,
-    "removeComments": true,
-    "resolveJsonModule": true,
-    "sourceMap": true,
-    "strictNullChecks": true,
-    "strictBindCallApply": true,
-    "allowJs": true
-  },
-  "include": [
-    "bin/**/*.*",
-    "lib/**/*.*",
-    "test/**/*.*"
-  ],
-  "exclude": [
-    "node_modules"
-  ]
-}
+    "compilerOptions": {
+      "module": "commonjs",
+      "target": "es5",
+      "outDir": "build",
+      "noImplicitAny": true,
+      "noUnusedParameters": true,
+      "noUnusedLocals": true,
+      "preserveConstEnums": true,
+      "removeComments": true,
+      "resolveJsonModule": true,
+      "sourceMap": true,
+      "strictNullChecks": true,
+      "strictBindCallApply": true,
+      "allowJs": true
+    },
+    "include": [
+      "bin/**/*.*",
+      "lib/**/*.*",
+      "test/**/*.*",
+      "config.js"
+    ],
+    "exclude": [
+      "node_modules"
+    ]
+}
```

Comment:
By changing all the whitespace on this file I can't see what actually changed.

Reply:
Things got messed up locally. I had to revert and stash quite a bit of change because first I intended to do a PR with full client update (check branch name) then I thought to break it up into logical chunks and then I thought ... actually thought too much with a weekend coming in. Let me reset and open a new PR. Begin again.